README.md

```diff
@@ -27,6 +28,22 @@ To reset the environment, clean up with `python manage.py clean`.
 6. Run the test suite with `python manage.py test`
 
 ## Deployment
-As an example of deployment, spkrepo provides [SaltStack](http://www.saltstack.com/) files in the `salt` directory.
-You can test this with [Vagrant](https://www.vagrantup.com/) by running `vagrant up`. This makes easy to reproduce
-production-like environments.
+
+https://flask.palletsprojects.com/en/1.1.x/tutorial/deploy/
+
+Modify `spkrepo/config.py` to disable debug logs, set a secure key and optionally set a cache:
+
+```env
+DEBUG = False
+TESTING = False
+SECRET_KEY = "secret-key"
+SQLALCHEMY_ECHO = False
+CACHE_TYPE="simple"
+```
+
+### Serve app via [a WSGI server](https://flask.palletsprojects.com/en/1.1.x/deploying/). Example:
+
+```shell
+pip install waitress
+waitress-serve --call 'spkrepo:create_app'
```

Comment:
spkrepo ships with a wsgi.py that defines app so I think you can do:
```suggestion
waitress-serve 'wsgi:app'
```

From within the directory.
I want to have a release process so one can `pip install spkrepo` and run it, probably the wsgi.py file won't be included there.